x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/grouping/GroupingFunction.java

```diff
@@ -26,23 +26,29 @@
 public abstract class GroupingFunction extends Function {
 
     private final Expression field;
+    private final List<Expression> extraChildren;
     private final List<Expression> parameters;
 
     private GroupingFunctionAttribute lazyAttribute;
 
-    protected GroupingFunction(Source source, Expression field) {
-        this(source, field, emptyList());
+    protected GroupingFunction(Source source, Expression field, List<Expression> extraChildren) {
```

Comment:
`extraChildren` is meaningless - use `arguments` instead (to preserve the terminology from the parent) or `parameters` instead.